lib/hammer_cli_csv/content_views.rb

```diff
@@ -9,6 +9,9 @@ class ContentViewsCommand < BaseCommand
       command_name 'content-views'
       desc         'import or export content-views'
 
+      option %w(--no-publish), :flag, _('Skip publishing content views on import')
+      option %w(--no-promote), :flag, _('Skip promoting content views on import')
```

Comment:
The `--export` validation sounds like a good idea.

I wasn't thinking that `--no-publish` requires `--no-promote`. IOW, if you pass in `--no-publish` it automatically doesn't promote. You can still pass in `--no-promote` but you don't have to. Similarly, if you call `--promote` I was thinking that it would automatically publish. I was thinking this would be more convenient for users but I suppose maybe explicit is better.